scripts/build_xmr-stak_docker/build_xmr-stak_docker.sh

```diff
@@ -5,110 +5,116 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
-if [ -d xmr-stak-cpu ]; then
-  git -C xmr-stak-cpu clean -fd
+if [ -d xmr-stak ]; then
+  git -C xmr-stak clean -fd
 else
-  git clone https://github.com/fireice-uk/xmr-stak-cpu.git
+  git clone https://github.com/fireice-uk/xmr-stak.git
 fi
 
 
 ########################
-# Fedora 26
+# Fedora 27
 ########################
-docker run --rm -it -v $PWD/xmr-stak-cpu:/xmr-stak-cpu fedora:26 /bin/bash -c "
+docker run --rm -it -v $PWD/xmr-stak:/xmr-stak fedora:27 /bin/bash -c "
 set -ex ;
 dnf install -y -q gcc gcc-c++ hwloc-devel libmicrohttpd-devel libstdc++-static make openssl-devel cmake ;
-cd /xmr-stak-cpu ;
-cmake -DCMAKE_LINK_STATIC=ON . ;
-make install ;
+cd /xmr-stak ;
+cmake -DCUDA_ENABLE=OFF -DOpenCL_ENABLE=OFF . ;
```

Comment:
why do you build without CUDA and OpenCL support?

CUDA and OpenCL  install is for all linux something like the following:
```
ENV PATH=/CUDA_ROOT=/usr/local/cuda-8.0

RUN  wget https://developer.nvidia.com/compute/cuda/8.0/prod/local_installers/cuda_8.0.44_linux-run && \
    chmod 755 ./cuda_8.0.44_linux-run && \
    ./cuda_8.0.44_linux-run --silent --toolkit --toolkitpath=$CUDA_ROOT && \
    rm ./cuda_8.0.44_linux-run
```

Reply:
I added CUDA wherever it was supported by the installer.